zephyr/module.yml

```diff
@@ -27,6 +27,14 @@ blobs:
     url: http://wiki.telink-semi.cn/wiki/protocols/Zephyr/binaries/public/tl321x_lib/lib_zephyr_tl321x_37251a7ca8f8488443f70dc6a09de19b604894a4.a
```

Comment:
Update hash of lib as well for lib_zephyr_tl321x.a
generally lib_zephyr_tl321x.a & lib_zephyr_tl321x_pm.a should have same build revision (this two libraries builded from same branch and published simultaneously) same as lib_zephyr_tl721x.a & lib_zephyr_tl721x_pm.a